src/mlpack/tests/ann_layer_test.cpp

```diff
@@ -2027,9 +2027,10 @@ TEST_CASE("BatchNormTest", "[ANNLayerTest]")
 
   // Value calculates using torch.nn.BatchNorm2d(momentum = None).
   arma::mat result;
-  result << 1.1658 << 0.1100 << -1.2758 << arma::endr
-         << 1.2579 << -0.0699 << -1.1880 << arma::endr
-         << 1.1737 << 0.0958 << -1.2695 << arma::endr;
+ 
```

Comment:
```suggestion
```
I don't think this new line is required. There are other places with an extra newline.